src/browser/api/window.js

```diff
@@ -35,6 +34,7 @@ import { toSafeInt } from '../../common/safe_int';
 import route from '../../common/route';
 import { FrameInfo } from './frame';
 import { System } from './system';
+import * as WebContents from './webcontents';
```

Comment:
an opinionated note -imo it's nicer when there's 1 default export containing an object with all the methods in it, along with individual exports for the methods. That way this line turns to 

> import WebContents from './webcontents';